lib/utils/get-auto-jsonc-rules-config/calculate-config-for-file.ts

```diff
@@ -0,0 +1,15 @@
+import type { Linter } from "eslint";
+// @ts-expect-error -- ignore
```

Comment:
What's the problem here?

Reply:
typescript gives an error saying `synckit` is an ESM package, but `synckit` can also be used with commonjs. I don't quite understand the correct way to configure typescript 😓

Reply:
I'll take a look and raise a PR for it.